### PR TITLE
Remove "experimental" in AMP stories tutorial

### PR DESCRIPTION
**Pull request by captain-lee**
AMP story has been launched.

### CODE REVIEW

**Comment by bpaduch** [comment]
Strange, but the component is still marked "Experimental" - https://www.ampproject.org/docs/reference/components/amp-story.

@newmuis - Can you confirm that amp-story is no longer experimental?

**Comment by bpaduch** [comment]
It looks like the "origin trial" is finished, but the component appears to still be "experimental".

**Comment by captain-lee** [comment]
Yes, it appears so according to this PR (https://github.com/ampproject/amphtml/pull/16619). Let's check again with @newmuis since it's been a month from the last comment.

**Comment by newmuis** [comment]
Although the origin trial is over, and people are free to publish stories without whitelisting of any kind, the component is still considered to be experimental.

**Comment by captain-lee** [comment]
Thanks for the clarifications, @newmuis and @bpaduch.

I brought back the "experimental" mark in visual_story.md file.

Please advise about setting_up.md as well, I intentionally didn't bring the following warning back because the tutorial uses amp-story v1.0 which doesn't state the message "You must enable the amp-story..." any more?
"You may encounter a screen that states "<b>You must enable the amp-story experiment to view this content</b>". The amp-story component is currently [experimental]..."

File: content/docs/design/visual_story/setting_up.md [hunk 1]
```diff
@@ -40,12 +40,6 @@ http://localhost:8000/pets-completed.html
 Make sure the URL serves from `localhost` otherwise the AMP story might not load correctly, and you may encounter errors like `"source" "must start with "https://" or "//" or be relative and served from either https or from localhost.`
 {% endcall %}
 
-{% call callout('Important', type='caution') %}
-You may encounter a screen that states "<b>You must enable the amp-story experiment to view this content</b>". The amp-story component is currently [experimental](/docs/reference/experimental.html).  To view the story, you must click the **Enable** button to turn on the amp-story component for your domain, and then reload the page. Although not necessary for this tutorial, you might want to opt your browser into the AMP Dev Channel, which allows your browser to use a newer, experimental version of the AMP JS libraries.  You can learn more about the [AMP Dev Channel here](/docs/reference/experimental.html).
```
Comment (bpaduch) [inline content/docs/design/visual_story/setting_up.md]: So, this isn't true anymore?  That is, you don't need to enable the "amp-story" experiment for stories? @newmuis

File: content/docs/design/visual_story.md [hunk 2]
```diff
@@ -19,8 +19,7 @@ In this tutorial, we'll introduce you to the [amp-story](/docs/reference/compone
 - Animate elements on a page
 - Keep readers engaged with your content by adding related links to the end of the story
 
-{% call callout('Note', type='note') %} The [amp-story](/docs/reference/components/amp-story.html) component is **experimental**. To sign up for the origin trial to publish pages with this component, please visit <a href="http://bit.ly/amp-story-signup">bit.ly/amp-story-signup</a>. {% endcall %}
-
+{% call callout('Note', type='note') %} The [amp-story](/docs/reference/components/amp-story.html) component is **experimental**. But, it's [open to all developers without origin trials or a whitelist](/latest/blog/new-in-amp-stories-monetization-revamped-bookends-and-metadata/). {% endcall %}
```
Comment (bpaduch) [inline content/docs/design/visual_story.md]: I'd recommend dropping any mention of origin or whitelist, simply keep the first sentence.

Comment (captain-lee) [reply content/docs/design/visual_story.md]: done.

**Comment by bpaduch** [comment]
@newmuis - To clarify, if the amp-story is experimental, do they still have to enable it (via the console, or via Dev Channel, or via Experimentals page) for the page to work?

**Comment by newmuis** [comment]
No, there are no special steps required to use `amp-story` when using `https://cdn.ampproject.org/v0/amp-story-1.0.js`.

**Comment by bpaduch** [comment]
Great!

**Pull request merged**